packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -681,7 +681,8 @@ class _AdbLogReader extends DeviceLogReader {
       _timeOrigin = null;
     runCommand(device.adbCommandForDevice(args)).then<void>((Process process) {
       _process = process;
-      const Utf8Decoder decoder = Utf8Decoder(allowMalformed: true);
+      // Previously this line specified allowMalformed: true.
```

Comment:
Avoid "history of the code" comments. Maybe explain why reportErrors is true, instead.